test/datastore-test.js

```diff
@@ -221,4 +221,68 @@ describe("datastore", () => {
       assert.strictEqual(result, ds);
     });
   });
+
+  describe("persistence", () => {
+    let cached = new Map();
+    let expectedID;
+    let something = {
+      title: "Sa Tuna2",
+      entry: {
+        kind: "login",
+        username: "foo",
+        password: "bar"
+      }
+    };
+
+    before(async () => {
+      localdatabase.startup();
+    });
+    after(async () => {
+      // cleanup databases
+      await localdatabase.teardown();
+    });
+
+    it("add a value to 1st datatore", async () => {
+      let main = new DataStore({
```

Comment:
It is better to construct `main` without any arguments, then call `.initialize()` just after `.prepare()`.